certmgr/mgr/file.go

```diff
@@ -201,6 +215,20 @@ func ReadSpecFile(path string, defaults *ParsableSpecOptions) (*cert.Spec, error
 	// transfer the parsed durations into their final resting spot.
 	spec.FinalizeSpecOptionParsing()
 
+	if len(spec.ParsedKeyUsages) > 0 {
+		for _, KeyUsageRaw := range spec.ParsedKeyUsages {
+			keyUsage, ok := config.ExtKeyUsage[strings.ToLower(KeyUsageRaw)]
+			if !ok {
+				log.Errorf("spec %s specifies unknown key usage '%s'. Valid values are: [%v]", path, KeyUsageRaw, validExtUsage)
```

Comment:
Instead of logging this error and moving on, should it be returned?

Reply:
Good catch, I'll push an update.

It's worth mentioning that this makes certmgr bail out of its initial directory scan even if there are other valid specs; but this is how we behave with other invalid configurations (e.g not being valid json), so we should do that here too.